runtime/gc_modron_startup/mminit.cpp

```diff
@@ -3140,6 +3141,9 @@ gcInitializeDefaults(J9JavaVM* vm)
 			break;
 		}
 
+		/* set vm->isVirtualLargeObjectHeapEnabled after extensions->isVirtualLargeObjectHeapEnabled is set in j9gc_initialize_heap(). */
+		vm->isVirtualLargeObjectHeapEnabled = (extensions->isVirtualLargeObjectHeapEnabled) ? TRUE : FALSE;
```

Comment:
it's cleaner if you can move it earlier, next to initializeIndexableObjectHeaderSizes call site 
this is a part of while loop that tries to create heap